cyral/resource_cyral_repository_test.go

```diff
@@ -12,13 +12,15 @@ var initialRepoConfig RepoData = RepoData{
 	Host:     "mongo.local",
 	Port:     3333,
 	RepoType: "mongodb",
+	Labels:   []string{"rds", "us-east-2"},
 }
 
 var updatedRepoConfig RepoData = RepoData{
 	Name:     "repo-test-updated",
 	Host:     "mongo-updated.local",
 	Port:     3334,
 	RepoType: "mongodb",
+	Labels:   []string{"rds", "us-east-1"},
```

Comment:
Please, add some test checks for the labels. You can use something like:
```
TestCheckResourceAttr("cyral_repository.test_repo_repository", "labels.#", "2")
```
To ensure that the two labels were set. For more info about check functions see:
https://www.terraform.io/plugin/sdkv2/testing/acceptance-tests/teststep#builtin-check-functions

Reply:
I add other 2 TestCheckResourceAttr, and read the docs about. Thanks for take some time to explain those details to me, help me a lot :)